src/index.js

```diff
@@ -32,6 +32,16 @@ function setupTimeoutCapture(wrapperInstance) {
 
 let invocationContext;
 
+function handlePromiseRejections(error) {
+  if (invocationContext && typeof invocationContext.fail === 'function') {
+    // default node behavior is to log these types of errors
+    console.error(error);
+    invocationContext.fail(error);
```

Comment:
Does Lambda fail the invocation on an uncaught promise error?

Reply:
Great question, it looks like "no" at the moment.

Reply:
We would fail invocations when aws doesn't? Not too fond of that, if true.